api/apps/articles/components/instant_articles/index.jade

```diff
@@ -20,10 +20,12 @@ html(lang="en" prefix="op: http://media.facebook.com/op#")
 
         h1= article.get('title')
 
-        if hero && article.strip(hero.deck)
-          h2= hero.deck
+        if hero && hero.deck
+          h2
+            i= hero.deck
         else if article.strip(leadParagraph)
-          h2!= leadParagraph
+          h2
+            i= article.strip(leadParagraph)
```

Comment:
I ran into these changes as I was running a backfill on Facebook Instant Articles. Just a minor thing I slipped in here.